lmdeploy/__init__.py

```diff
@@ -1,4 +1,10 @@
 # Copyright (c) OpenMMLab. All rights reserved.
+
+from lmdeploy.api import client, pipeline, serve
+
+__all__ = ['pipeline', 'serve', 'client']
```

Comment:
If we put `serve` here, should we add the serve dependencies to runtime.txt?

Reply:
No idea here. @RunningLeon Any comments?

Reply:
`pipeline`, `serve`, `client` all involve codes from `lmdeploy.serve`. Maybe we add a note in docs to let users install lmdeploy by `pip install lmdeploy[serve]` when using this api?